strawberryfields/apps/qchem/utils.py

```diff
@@ -96,3 +96,59 @@ def duschinsky(
     delta = np.array(d @ l0_inv)
 
     return U, delta
+
+
+def read_gamess(file) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    r"""Reads molecular data from the output file generated by the GAMESS quantum chemistry package.
+
+    This function extracts the atomic coordinates, atomic masses, vibrational frequencies,
+    and normal modes of a molecule from the output file of a vibrational frequency calculation
+    performed with the GAMESS quantum chemistry package.
+
+    **Example usage:**
+
+    >>> r, m, w, l = read_gamess('../BH_frq.out')
```

Comment:
Maybe nice to print each of these four with a short comment explaining what they stand for? Would also be good to give the `.out` file a clearer name. For example
```
>>> r, m, w, l = read_gamess('[molecule_name]_data.out'
>>> r  # atomic coordinates
array([0.000000, 0.000000, 0.000000, 1.253604, 0.000000, 0.000000]
```